starcheck/data/ACABadPixels

```diff
@@ -1,255 +1,3 @@
-Bad Pixel List from 2017272 2018002 2018100 Dark Currents
-
 Star.Body.Pixels.BadPixels = [ ...
- -512,  511,   -1,    0; ...
-   -1,    0, -512,  511; ...
  -245,    0,  454,  454; ...
```

Comment:
The intent here is that only the large CCD trap remains (not sure if the fid defect should be added).

Functionally I just checked that an observation that previously had a star with a trap-related warning still did so with this bad pixel list.  Obsid 48812 in OCT2918A
```
>> WARNING: [ 8] Nearby ACA bad pixel.  row, col (-64, 454), dy, dz (0, 6) 
```

Reply:
I did not add a comment in the file about the trap as I didn't want to play games with the parser.